upstream.go

```diff
@@ -204,9 +204,30 @@ func (uc *upstreamConn) handleMessage(msg *irc.Message) error {
 	case "NOTICE":
 		uc.logger.Print(msg)
 
-		uc.forEachDownstream(func(dc *downstreamConn) {
-			dc.SendMessage(msg)
-		})
+		if msg.Prefix.User == "" && msg.Prefix.Host == "" { // server message
```

Comment:
Can the IRC server send a NOTICE without any prefix? This is what Freenode sends for a server NOTICE:

```
:verne.freenode.net NOTICE * :*** Looking up your hostname...
```

Reply:
in that case, `msg.Prefix.Name = verne.freenode.net`, `msg.Prefix.User = ""`, `msg.Prefix.Host = ""`. that's why i used `msg.Prefix.User == "" && msg.Prefix.Host == ""`

Reply:
Ah, got it!